.github/workflows/pull-request.yml

```diff
@@ -11,11 +11,14 @@ defaults:
 permissions:
   contents: read
   actions: read
+  pull-requests: write
 
 env:
   TURBO_REF_FILTER: ${{ github.event_name == 'pull_request' && github.event.pull_request.base.sha || github.event.before }}
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
   TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+  VERCEL_TOKEN: ${{ secrets.TURBO_TOKEN }}
```

Comment:
is this value used if set by one of the actions, such as zentered/vercel-preview-url ? I don't see it referenced anywhere directly

Reply:
yes https://github.com/zentered/vercel-preview-url#environment-variables--secret

Reply:
Yes, used by `zentered/vercel-preview-url`. It's actually being referred down there :)